halo/attest/voter/voter.go

```diff
@@ -191,6 +191,7 @@ func (v *Voter) runOnce(ctx context.Context, chainVer xchain.ChainVersion) error
 		Offset:    fromOffset,
 	}
 
+	cursors := make(map[xchain.StreamID]uint64, len(v.network.Chains))
```

Comment:
The relayer's `msgCursorFilter` that I based this off has a dedicated struct for the entries in this map (see `streamCursor` [here](https://github.com/omni-network/omni/blob/3aa61db7c4295185cb3a1f9bed5d1ce483ed72de/relayer/app/cursors.go#L131-L133)). I wasn't sure if this was considered better practice, and didn't want to blindly copy it, so I just have a `uint64`. 

Feedback welcome!

Reply:
this is great #lessismore